services/vqa-service/service_pb2.py

```diff
@@ -0,0 +1,167 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Autogenerated file should not be kept in repo, I would create `setup.py` which builds Python package using protobuf compiler instead.